templates/core/build.html

```diff
@@ -5,151 +5,180 @@
+        <p class="p-card__content">Qualcomm DragonBoard</p>
+      </button>
     </div>
-    <div class="col-6 p-divider__block">
-      <h4 class="p-muted-heading">Pre-installed snaps</h4>
-      <hr class="u-sv1">
-      <div class="js-preinstalled-snaps-list"></div>
+  </section>
+
+  <section class="p-strip is-shallow">
+    <div class="row">
+      <div class="col-8">
+        <h2>2 What version of Ubuntu do you want?</h2>
+      </div>
     </div>
-  </div>
-</section>
+    <div class="row js-os">
+      <button class="col-2 p-card js-selection" data-supports="raspberry-pi2 raspberry-pi3 intel-nuc qualcomm-dragonboard">
+        <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/c4f35e06-products-hero-ubuntu.svg" alt="Ubuntu logo">
+        <hr class="u-sv1"/>
+        <p class="p-card__content">Core 16</p>
+      </button>
+      <button class="col-2 p-card js-selection" data-supports="raspberry-pi2 raspberry-pi3 raspberry-pi4 intel-nuc qualcomm-dragonboard">
+        <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/c4f35e06-products-hero-ubuntu.svg" alt="Ubuntu logo">
+        <hr class="u-sv1"/>
+        <p class="p-card__content">Core 18</p>
+      </button>
+      <button class="col-2 p-card js-selection" data-supports="raspberry-pi2 raspberry-pi3">
+        <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/c4f35e06-products-hero-ubuntu.svg" alt="Ubuntu logo">
+        <hr class="u-sv1"/>
+        <p class="p-card__content">Classic 16.04</p>
+      </button>
+      <button class="col-2 p-card js-selection" data-supports="raspberry-pi2 raspberry-pi3 raspberry-pi4">
+        <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/c4f35e06-products-hero-ubuntu.svg" alt="Ubuntu logo">
+        <hr class="u-sv1"/>
+        <p class="p-card__content">Classic 18.04</p>
+      </button>
+      <button class="col-2 p-card js-selection" data-supports="raspberry-pi3 raspberry-pi4">
+        <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/c4f35e06-products-hero-ubuntu.svg" alt="Ubuntu logo">
+        <hr class="u-sv1"/>
+        <p class="p-card__content">Classic 64-bit 18.04</p>
+      </button>
+    </div>
+  </section>
 
-<section class="p-strip--light is-shallow">
-  <div class="row">
-    <h2>Summary</h2>
-  </div>
-  <div class="row p-divider">
-    <div class="col-6 p-divider__block">
-      <dl>
-        <dt>Board</dt>
-        <dd class="js-summary-board"></dd>
-        <dt>Operating system</dt>
-        <dd class="js-summary-os"></dd>
-        <dt>Snaps</dt>
-        <dd><span class="js-summary-snaps">0</span> pre-installed snaps</dd>
-      </dl>
+  <section class="p-strip is-shallow">
+    <div class="row">
+      <div class="col-8">
+        <h2>3 What snaps do you want preinstalled?</h2>
+        <p>Select the applications that will be pre-installed on the device. For Ubuntu Core this will be snaps, for Ubuntu Classic it will be snaps or packages.</p>
```

Comment:
I didn’t know this. We’ll need to figure out how to distinguish them when there’s a snap and a .deb package with the same name.

Snaps are a kind of package, so “or packages” doesn’t really work. I think you need to specify “.deb packages”.

Reply:
I stole this from [the prototype](https://image.iotdevice.io/boards/raspberrypi2/core16). This might not be the case if we only support core.

Reply:
Not for this PR, but: If .deb packages are an open question, or something that might change later, that suggests `core` maybe shouldn’t be baked into the URL. (Another reason is that this page would be quite at home in ubuntu.com’s IoT section.)